### PR TITLE
keep uuids unique when copying an app

### PR DESCRIPTION
**Pull request by nickpell**
fix for duplicate reports appearing after copying an app: http://manage.dimagi.com/default.asp?183786

@orangejenny

### CODE REVIEW

File: corehq/apps/app_manager/models.py [hunk 1]
```diff
@@ -5152,6 +5152,12 @@ def import_app(app_id_or_source, domain, source_properties=None, validate_source
         del source['build_spec']
     app = cls.from_source(source, domain)
     app.cloudcare_enabled = domain_has_privilege(domain, privileges.CLOUDCARE)
+
+    for module in app.get_modules():
+        if isinstance(module, ReportModule):
+            for report_config in module.report_configs:
+                report_config.uuid = 'copy-' + report_config.uuid
```
Comment (orangejenny) [inline corehq/apps/app_manager/models.py]: is this potentially going to make ids like `copy-copy-copy-foo`? why not generate a new uuid?

Comment (nickpell) [inline corehq/apps/app_manager/models.py]: this is possible - I like keeping the original uuid around, and didn't want to implement logic for ```copy-#-```.  I don't think it's super likely that a user will copy an app enough times for this to become a problem.

Comment (orangejenny) [inline corehq/apps/app_manager/models.py]: Fair enough*  Will merge once tests pass.

*although, did you know, we have code that's written to help with this problem: https://github.com/dimagi/dimagi-utils/blob/master/dimagi/utils/next_available_name.py ...but you'd still need to write code to gather all of the existing reports that include the uuid, so it's not quite trivial, understand if it isn't worth the effort

Comment (nickpell) [inline corehq/apps/app_manager/models.py]: That's really good to know ... although I just realized that either approach would produce duplicates if two copies are made of the original app ... so I will fix tomorrow.  new uuid it is.

Comment (nickpell) [inline corehq/apps/app_manager/models.py]: fixed - always creates a new uuid

**Pull request merged**